security/ee-security-basic/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2015 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+  -->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.thorntail.examples</groupId>
+    <artifactId>examples-jaxrs</artifactId>
+    <version>2.3.1.Final-SNAPSHOT</version>
+    <relativePath>../</relativePath>
+  </parent>
+
+  <artifactId>example-ee-security-basic</artifactId>
+
+  <name>Thorntail Examples: Java EE 8 Security - Basic Authentication</name>
+  <description>Thorntail Examples: Java EE 8 Security - Basic Authentication</description>
+
+  <packaging>war</packaging>
+
+  <properties>
+    <version.resteasy>3.0.14.Final</version.resteasy>
```

Comment:
This should probably be 3.6.2.Final these days.

Reply:
Fixed.